File: commons/src/main/java/com/eirsteir/coffeewithme/commons/domain/coffeebreak/CoffeeBreakCreatedEvent.java
package com.eirsteir.coffeewithme.commons.domain.coffeebreak;

import com.eirsteir.coffeewithme.commons.domain.notification.AbstractEntityNotificationEvent;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class CoffeeBreakCreatedEvent  extends AbstractEntityNotificationEvent implements CoffeeBreakEvent {

    private CoffeeBreakDetails coffeeBreakDetails;

}

File: commons/src/main/kotlin/com/eirsteir/coffeewithme/commons/domain/coffeebreak/CoffeeBreakCreatedEvent.kt
package com.eirsteir.coffeewithme.commons.domain.coffeebreak

import com.eirsteir.coffeewithme.commons.domain.notification.AbstractEntityNotificationEvent
import com.eirsteir.coffeewithme.commons.domain.user.UserDetails

open class CoffeeBreakCreatedEvent(
    val coffeeBreakDetails: CoffeeBreakDetails,
) : AbstractEntityNotificationEvent(), CoffeeBreakEvent